components/script/dom/htmlscriptelement.rs

```diff
@@ -1010,10 +1008,21 @@ impl HTMLScriptElement {
 
         if let Some(chan) = self.global().devtools_chan() {
             let pipeline_id = self.global().pipeline_id();
+
+            let content = match &script.code {
+                SourceCode::Text(text) => text.to_string(),
+                SourceCode::Compiled(compiled) => compiled.original_text.to_string(),
+            };
```

Comment:
we found that for inline scripts, this sends only the script contents, but we should be sending the whole HTML file. see #36874